tests/libpod/render.go

```diff
@@ -198,3 +209,32 @@ func RenderTargetcliPod(name, disksPVC string) *v1.Pod {
 		},
 	}
 }
+
+func AddServiceBackHoleForKubernetesApiToPod(pods *v1.PodList, virtCli kubecli.KubevirtClient, componentName string) {
+	serviceBlackHoleForKubernetesApiToPod(pods, virtCli, componentName, BlackHoleActionAdd)
+}
+
+func DeleteServiceBlackHoleForKubernetesApiToPod(pods *v1.PodList, virtCli kubecli.KubevirtClient, componentName string) {
+	serviceBlackHoleForKubernetesApiToPod(pods, virtCli, componentName, BlackHoleActionDel)
+}
+
+func serviceBlackHoleForKubernetesApiToPod(pods *v1.PodList, virtCli kubecli.KubevirtClient, componentName string, addOrDel BlackHoleAction) {
```

Comment:
Instead of `addOrDel` you could make it a bool called `present`.

`present == true` --> add the blackhole
`present == false` --> remove the blackhole

Reply:
Done.